### PR TITLE
Update config values

### PR DESCRIPTION
**Pull request by sczembor**
+ update the config values
+ add `get_max_turn_duration` method

### CODE REVIEW

File: contracts/cheddar_3x3_tic_tac_toe/src/lib.rs [hunk 1]
```diff
@@ -99,7 +99,7 @@ impl Contract {
                 MAX_FEES,
                 // 1 hour for max_game_duration will be set
                 // also 400 sec will be max turn duration (max_game_duration / MAX_TURNS_NUM)
-                sec_to_nano(60 * 60),
+                sec_to_nano(5 * 60),
```
Comment (robert-zaremba) [inline contracts/cheddar_3x3_tic_tac_toe/src/lib.rs]:
```suggestion
                sec_to_nano(MAX_GAME_DURATION_SEC),
```

File: contracts/cheddar_3x3_tic_tac_toe/src/config.rs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 use crate::*;
 
 /// 1 HOUR in seconds
-const MAX_GAME_DURATION_SEC: u32 = 60 * 60;
+const MAX_GAME_DURATION_SEC: u32 = 5 * 60;
 const MIN_GAME_DURATION_SEC: u32 = 100;
```
Comment (robert-zaremba) [inline contracts/cheddar_3x3_tic_tac_toe/src/config.rs:5]: why do we need `MIN_GAME_DURATION_SEC`?

File: contracts/cheddar_5x5_tic_tac_toe/src/views.rs [hunk 3]
```diff
@@ -178,7 +178,10 @@ impl Contract {
...
         result
     }
-    pub fn get_max_game_duration(&self) -> u64 {
-        return self.max_game_duration;
+    pub fn get_max_game_duration(&self) -> u32 {
+        return nano_to_sec(self.max_game_duration).into();
+    }
+    pub fn get_max_turn_duration(&self) -> u32 {
+        return nano_to_sec(self.max_turn_duration.into());
     }
```
Comment (robert-zaremba) [inline contracts/cheddar_5x5_tic_tac_toe/src/views.rs]: let's remove these functions, and use the values in config.

**Review by robert-zaremba — APPROVED** [review]
pre-approving

File: contracts/cheddar_3x3_tic_tac_toe/src/config.rs (hunk 2 above)
Comment (sczembor) [reply contracts/cheddar_3x3_tic_tac_toe/src/config.rs:5]: game duration is being checked against it when initialising config

File: contracts/cheddar_5x5_tic_tac_toe/src/views.rs (hunk 3 above)
Comment (sczembor) [reply contracts/cheddar_5x5_tic_tac_toe/src/views.rs]: done

**Pull request merged**